lib/api.js

```diff
@@ -56,7 +61,11 @@ function end(id, options = {}) {
     ;
 
   return Promise
-          .resolve(fetch(uri, opts))
+          .try(() => {
+            return retry(() => {
+              return fetch(uri, opts);
+            }, { max_tries: 5, interval: 1000 });
+          })
```

Comment:
no reason to nest `retry` in in the `try`

just do:
```
return retry(() => {
              return fetch(uri, opts);
            }, { max_tries: 5, interval: 1000 })
            .then(...
```

change for all of them